bluejeans/packages/bluejeans/bluejeans_list.js

```diff
@@ -86,8 +82,8 @@ async function _command(params = {}, commandText, secrets = {}) {
  * @property {string} text
  * @property {'in_channel'|'ephemeral'} [response_type]
  */
-const main = async (args) => ({
-  body: await _command(args.params, args.commandText, args.__secrets || {}).catch(error => ({
+const main = async ({params = {}, commandText, __secrets = {}}) => ({
```

Comment:
After multi-language support accessing prams, commandText and _secrets is not working in this way. Use the args while defining the main and use args.params and so on when you are passing to _command.

also using args.__secrets || {} should be fine. I don't know if we are not adding any secrets then this will be null or empty if its null then assiging with {} will be better. @niks3089  can you update us on it?

Reply:
We're [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) the args and using function [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) to assign an empty object if the value is undefined.

Reply:
Yes it is fine when you are using `const main = async ({params = {}, commandText, __secrets = {}})` but this way I was facing the issues after MultiLanguage support that's why we changed it to

async (args) and used as args.params.  And only after reverting to async(args) I was concerned for args.__secrets || {} . 

Are you not facing the issue with `const main = async ({params = {}, commandText, __secrets = {}})`? Did you tried after updating node_modules(updating the dev dependecies).

Reply:
The only change with multi-language support was to avoid "spill-over" of params so we can use other languages. However, this may work but I would recommend having the main be same/consistent across the command-sets

Reply:
Which one do you prefer? I feel like `const main = async ({params = {}, commandText, __secrets = {}})` is more modern and also uses the intended features of the language like default parameters.

Reply:
I prefer the `main(args)` since that's usually a standard in other OW serverless functions as well in the other languages.

Reply:
> Yes it is fine when you are using `const main = async ({params = {}, commandText, __secrets = {}})` but this way I was facing the issues after MultiLanguage support that's why we changed it to
> 
> async (args) and used as args.params. And only after reverting to async(args) I was concerned for args.__secrets || {} .
> 
> Are you not facing the issue with `const main = async ({params = {}, commandText, __secrets = {}})`? Did you tried after updating node_modules(updating the dev dependecies).

No, I didn't face any issues. They both are essentially the same.